diff --git a/node_modules/jest-expect-message/dist/withMessage.js b/node_modules/jest-expect-message/dist/withMessage.js
index 48de706..7af6e38 100644
--- a/node_modules/jest-expect-message/dist/withMessage.js
+++ b/node_modules/jest-expect-message/dist/withMessage.js
@@ -8,7 +8,7 @@ var _extends = Object.assign || function (target) { for (var i = 1; i < argument
 
 class JestAssertionError extends Error {
   constructor(result, callsite) {
-    super(result.message());
+    super(typeof result.message === 'function' ? result.message() : result.message);
     this.matcherResult = result;
 
     if (Error.captureStackTrace) {
@@ -31,7 +31,10 @@ const wrapMatcher = (matcher, customMessage) => {
         throw new JestAssertionError(matcherResult, newMatcher);
       }
 
-      const message = () => 'Custom message:\n  ' + customMessage + '\n\n' + matcherResult.message();
+      const matcherMessage =
+        typeof error.matcherResult.message === 'function' ? error.matcherResult.message() : error.matcherResult.message;
+
+      const message = () => 'Custom message:\n  ' + customMessage + '\n\n' + matcherMessage;
 
       throw new JestAssertionError(_extends({}, matcherResult, { message }), newMatcher);
     }
